core/src/main/java/com/linecorp/armeria/common/logging/RequestOnlyLog.java

```diff
@@ -84,6 +84,31 @@ public interface RequestOnlyLog extends RequestLogAccess {
     @Nullable
     Long requestFirstBytesTransferredTimeNanos();
 
+    /**
+     * Returns the time when the request is fully received, in microseconds since the epoch.
+     *
+     * @throws RequestLogAvailabilityException if the property is not available yet.
+     * @see RequestLogProperty#REQUEST_FULLY_RECEIVED_TIME
+     */
+    long requestFullyReceivedTimeMicros();
+
+    /**
+     * Returns the time when the request is fully received, in milliseconds since the epoch.
+     *
+     * @throws RequestLogAvailabilityException if the property is not available yet.
+     * @see RequestLogProperty#REQUEST_FULLY_RECEIVED_TIME
+     */
+    long requestFullyReceivedTimeMillis();
+
+    /**
+     * Returns the time when the request is fully received, in nanoseconds. This value can only be
+     * used to measure elapsed time and is not related to any other notion of system or wall-clock time.
+     *
+     * @throws RequestLogAvailabilityException if the property is not available yet.
+     * @see RequestLogProperty#REQUEST_FULLY_RECEIVED_TIME
+     */
+    long requestFullyReceivedTimeNanos();
```

Comment:
I think this method is enough because this property is mostly used with the metric.
Could you remove above two methods, please?